lib/tags.js

```diff
@@ -87,19 +88,25 @@ exports.parseTagLine = (line) => {
  * console.log(footer.Bar);
  * > baz
  */
-exports.parseFooterTagLines = (footerTagLines) => {
+exports.parseFooterTagLines = (footerTagLines, lowerCaseFooterTags) => {
   return _.chain(footerTagLines)
     .reject((line) => {
       return _.isEmpty(line.trim());
     })
     .map((tagLine) => {
       const tag = exports.parseTagLine(tagLine);
+      const finalTags = [];
 
-      return [
-        tag.key,
-        tag.value
-      ];
+      finalTags.push([ tag.key, tag.value ]);
```

Comment:
Oh, I see now that your intention is to return both of them, so my comments above don't apply very much. I'll update them in a minute...